### ISSUE TITLE
Inquiries about yolov3 and yolov3-tiny model accuracy 

### ISSUE THREAD

**Issue by cig2982** [AlexeyAB/darknet#8078]
Hi 
I discovered a strange thing while measuring mAP on the coco validation set using the yolov3 and yolov3-tiny models. 

For yolov3, 66.19% is measured for the validation set as shown in the picture below. 
![yolov3](https://user-images.githubusercontent.com/50826339/132818790-443fff17-5a99-4ab7-9c36-13f2f6deac93.png)
This is a higher result than the result measured at 55.3% for the test set specified on the https://pjreddie.com/darknet/yolo/ homepage. (And is 66.19% too high compared to 55.3%? )

However, for yolov3-tiny, 19.41% is measured as shown in the figure below, which is very small compared to 33.1% measured for the test set. 
![yolov3-tiny](https://user-images.githubusercontent.com/50826339/132819127-edc61a76-625e-4b25-958c-054225ed00b9.png)
In my opinion, yolov3-tiny should also have a higher mAP measured on the validation set than on the test set. 
Please tell me why this is happening. Maybe you uploaded the wrong yolov3-tiny model? 

We would appreciate it if you could respond to the above.

**Comment by lsd1994** [comment]
What command line do you use?

**Comment by cig2982** [comment]
I used the below command

```
./darknet detector map ~
```

**Comment by cig2982** [comment]
> What command line do you use?
@lsd1994 
I used the below command
```
./darknet detector map *.data *.cfg *.weights
```

**Comment by louielu1027** [comment]
@cig2982  Do you know the reason ? ？ I got the same results.......when I test on testdev2017,  yolov3-tiny only got  map@0.5  0.183.  
scores.txt :
AP: 0.088
AP_50: 0.183
AP_75: 0.075
AP_small: 0.001
AP_medium: 0.055
AP_large: 0.227

**Comment by louielu1027** [comment]
1. change mask = 0,1,2 to mask = 1,2,3.
2. add letter_box=1 in [net]

but i dont know why.........

**Comment by mwessley** [comment]
> 1. change mask = 0,1,2 to mask = 1,2,3.
> 
>     2. add letter_box=1 in [net]
> 
> 
> but i dont know why......

it seems that back then the original TinyYolov3 weights you can download were trained on mask = 1,2,3 than later on the "mistake" was corrected in the mask but the weights were never updated.
